Assets/HoloToolkit/BuildAndDeploy/Editor/BuildDeployWindow.cs

```diff
@@ -315,41 +335,86 @@ private void OnGUI()
             }
 
             string newMSBuildVer = currentSDKVersionIndex <= defaultMSBuildVersionIndex ? BuildDeployTools.DefaultMSBuildVersion : "15.0";
-            EditorGUILayout.LabelField(GUIHorizSpacer + "MS Build Version", newMSBuildVer);
+            EditorGUILayout.LabelField(GUIHorizontalSpacer + "MS Build Version", newMSBuildVer);
 
             if (!newMSBuildVer.Equals(curMSBuildVer))
             {
                 BuildDeployPrefs.MsBuildVersion = newMSBuildVer;
                 curMSBuildVer = newMSBuildVer;
             }
 
+            GUILayout.EndHorizontal();
+            GUILayout.BeginHorizontal();
+
             // Build config (and save setting, if it's changed)
             string curBuildConfigString = BuildDeployPrefs.BuildConfig;
 
             BuildConfigEnum buildConfigOption;
             if (curBuildConfigString.ToLower().Equals("master"))
             {
-                buildConfigOption = BuildConfigEnum.MASTER;
+                buildConfigOption = BuildConfigEnum.Master;
             }
             else if (curBuildConfigString.ToLower().Equals("release"))
             {
-                buildConfigOption = BuildConfigEnum.RELEASE;
+                buildConfigOption = BuildConfigEnum.Release;
             }
             else
             {
-                buildConfigOption = BuildConfigEnum.DEBUG;
+                buildConfigOption = BuildConfigEnum.Debug;
             }
 
-            buildConfigOption = (BuildConfigEnum)EditorGUILayout.EnumPopup(GUIHorizSpacer + "Build Configuration", buildConfigOption);
+            buildConfigOption = (BuildConfigEnum)EditorGUILayout.EnumPopup(GUIHorizontalSpacer + "Build Configuration", buildConfigOption);
 
-            string newBuildConfig = buildConfigOption.ToString();
+            string buildConfigString = buildConfigOption.ToString();
 
-            if (newBuildConfig != curBuildConfigString)
+            if (buildConfigString != curBuildConfigString)
             {
-                BuildDeployPrefs.BuildConfig = newBuildConfig;
-                curBuildConfigString = newBuildConfig;
+                BuildDeployPrefs.BuildConfig = buildConfigString;
+                curBuildConfigString = buildConfigString;
             }
 
+            // Build Platform (and save setting, if it's changed)
+            string curBuildPlatformString = BuildDeployPrefs.BuildPlatform;
+
+            BuildPlatformEnum buildPlatformOption;
```

Comment:
How about something like this instead here:  
```cs
BuildPlatformEnum buildPlatformOption;
if (!Enum.TryParse(curBuildPlatformString, true /*ignoreCase*/, out buildPlatformOption))
{
    buildPlatformOption = BuildPlatformEnum.AnyCPU;
}
```

Reply:
The problem is that the `AnyCPU` enum string doesn't match the string we need to match with.

Reply:
But in that case, it will fail the TryParse and then be set inside the if branch. Granted, that behavior isn't spelled out directly.

Reply:
I don't see `System.Enum.TryParse`

Reply:
Yeah, looks like it was added in 4.0... bummer.